src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -277,12 +329,83 @@ private AzureFileSystem GetAzureFileSystem()
             return fileSystem;
         }
 
-        private async Task<bool> PushAsync(IEnumerable<string> items, bool allowOverwrite)
+        private async Task<bool> PushAsync(
+            IEnumerable<string> items,
+            bool allowOverwrite,
+            bool passIfExistingItemIdentical)
```

Comment:
Not a blocker, but `allowOverwrite` and `passIfExistingItemIdentical` seem like ideal candidates for flags on some `PackageUploadOptions` object.

Reply:
Agreed, created `PushOptions`. Not shared with `Microsoft.DotNet.Build.CloudTestTasks` `UploadToAzure`--there, these params are isolated to the task implementation so the grouping wouldn't have any benefit anyway.